pkgs/by-name/ht/hterm/package.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv
+, lib
+, fetchurl
+, cairo
+, pango
+, libpng
+, expat
+, fontconfig
+, gtk2
+, xorg
+, autoPatchelfHook
+, copyDesktopItems
+}:
+
+stdenv.mkDerivation
+  (finalAttrs: {
+    pname = "hterm";
+    version = "0.8.9";
+
+    src = fetchurl {
+      url = "https://www.der-hammer.info/terminal/hterm${builtins.replaceStrings ["."] [""] finalAttrs.version}-linux-${if stdenv.targetPlatform.is64bit then "64" else "32"}.tgz";
+      hash = "sha256-DY+X7FaU1UBbNf/Kgy4TzBZiocQ4/TpJW3KLW1iu0M0=";
```

Comment:
The same hash for both 64 and 32?

Reply:
No, I just forgot to add the hash for the 32-bit archive.